### PR TITLE
[MERGE AFTER RELEASE-2.1 BRANCH CREATED] Add release notes for v2.1.0

### PR DESCRIPTION
**Pull request by denyeart**
Add release notes for v2.1.0

Signed-off-by: David Enyeart <enyeart@us.ibm.com>

### CODE REVIEW

File: release_notes/v2.1.0.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+- Bump docker images to Alpine 3.11.
+- Bump Go to 1.14.1.
+- Bump Go grpc to 1.28.0.
+
+
+Fixes
+-----
+
+**FAB-17458: Rolling upgrade: Different endorsement results on v1.4.x and v2.0.0 peers (FIX ALSO IN V2.0.1)**
```
Comment (denyeart) [inline release_notes/v2.1.0.md:62]: For the fixes also in v2.0.1 (release-2.0 branch), I think I'll remove from the v2.1 release notes and just mention that all fixes in v2.0.1 are also merged into v2.1.

**Comment by robmurgai** [comment]
Should the line: Limits can now be placed on the number of **_endorser_** and deliver requests, be Limits can now be placed on the number of **_endorse_** and deliver requests ?

**Comment by denyeart** [comment]
> Should the line: Limits can now be placed on the number of **_endorser_** and deliver requests, be Limits can now be placed on the number of **_endorse_** and deliver requests ?

No, in the yaml it is known as endorserService and deliverService.

**Comment by denyeart** [comment]
PR for release-2.1 is https://github.com/hyperledger/fabric/pull/1001

**Pull request closed without merging**